gnomad_qc/v3/annotations/generate_sv_hists.py

```diff
@@ -171,12 +166,19 @@ def main(args):
     logger.info("Checkpoint SV MT...")
     mt = mt.annotate_cols(**sv_list[mt.col_key])
     mt = mt.checkpoint(temp_gnomad_sv_mt_path, overwrite=args.overwrite)
+    logger.info(
+        "Generating age and GQ histograms for %s variants and %s samples...",
```

Comment:
I didn't think about this last time, but should these be `%i`? (this doesn't really matter either way though)

Reply:
Yeah they get casted either way but consistency is always good!